--- x264.c	2020-02-23 18:14:20.086270112 +0100
+++ x264.c	2020-02-23 18:15:15.627178997 +0100
@@ -345,7 +345,7 @@
 #if HAVE_FFMS
     printf( "(ffmpegsource %d.%d.%d.%d)\n", FFMS_VERSION >> 24, (FFMS_VERSION & 0xff0000) >> 16, (FFMS_VERSION & 0xff00) >> 8, FFMS_VERSION & 0xff );
 #endif
-    printf( "built on " __DATE__ ", " );
+    printf( "built on unknown (bazel), " );
 #ifdef __INTEL_COMPILER
     printf( "intel: %.2f (%d)\n", __INTEL_COMPILER / 100.f, __INTEL_COMPILER_BUILD_DATE );
 #elif defined(__GNUC__)
